polymorphic/formsets/utils.py

```diff
@@ -1,11 +1,17 @@
 """
 Internal utils
 """
+from polymorphic.compat import DJANGO_GTE_2
 
 
 def add_media(dest, media):
     """
     Optimized version of django.forms.Media.__add__() that doesn't create new objects.
+
+    Only required for Django < 2.0
     """
-    dest.add_css(media._css)
-    dest.add_js(media._js)
+    if DJANGO_GTE_2:
+        dest += media
```

Comment:
⚠️  This no longer updates the `dest` object so it will break. Returning `dest` and updating `polymorphic/admin/inlines.py`, `polymorphic/formsets/models.py` would be better.

Reply:
Are you 100% sure this fails? I thought I had tested this quite thoroughly.

If you're sure you'd like this to change, please confirm, and I'll get it done.